src/main/java/com/kodilla/ecommercee/controller/OrderController.java

```diff
@@ -1,35 +1,56 @@
 package com.kodilla.ecommercee.controller;
 
-import com.kodilla.ecommercee.domain.GenericEntity;
+import com.kodilla.ecommercee.domain.*;
+import com.kodilla.ecommercee.mapper.OrderMapper;
+import com.kodilla.ecommercee.service.OrderService;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.web.bind.annotation.*;
-
-import java.util.ArrayList;
 import java.util.List;
 
 @RestController
 @RequestMapping("/v1/order")
+@AllArgsConstructor
 public class OrderController {
+
+    @Autowired
+    private final OrderService orderService;
+    @Autowired
+    private final OrderMapper orderMapper;
+
     @GetMapping(value = "getOrders")
-    public List<GenericEntity> getOrders() {
-        return new ArrayList<>();
+    public List<OrderDto> getOrders() {
+        List<Order> orders = orderService.getAllOrders();
+        return orderMapper.mapToOrderDtoList(orders);
     }
 
     @PostMapping(value = "createOrder")
-    public void createOrder(GenericEntity genericEntity) {
+    public OrderDto createOrder(long cardId, long userId) throws CartNotFoundException, UserNotFoundException {
```

Comment:
@RequestParam ?

Reply:
a czy to nie jest to z czego rezygnowaliśmy przy wstępnej implementacji?

Reply:
chyba nie, tamto było inne ale nie pamiętam już co to było

Reply:
ok to dorzucę